lib/semantics/check-do.cc

```diff
@@ -404,6 +418,7 @@ class DoContext {
     CheckDoExpression(bounds.upper);
     if (bounds.step) {
       CheckDoExpression(bounds.step.value());
+      CheckZeroStep(bounds.step.value());
```

Comment:
After checking for the presence of a value immediately beforehand, you don't have to use `.value()`.

Reply:
Thanks.  I'll change this.